### PR TITLE
Update imagemagick to 6.9.6-5

### PR DESCRIPTION
**Pull request by y0ssar1an**
The `imagemagick` package needed to be updated with the security fixes in libpng 1.6.26.

Update libpng to 1.6.26
Replace libjpeg with libjpeg-turbo (drop-in replacement recommended by libwebp)
Update giflib to 5.1.4
Update libtiff to 4.0.7
Update libwebp to 0.5.1
Add yasm dependency of libjpeg-turbo
Remove imagemagick 6.8 script from `packages/`

@variadico @zquestz @earlruby

### CODE REVIEW

File: packages/imagemagick-6.9.6-5.conf [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  { url: "https://s3.amazonaws.com/apcera-sources/imagemagick/giflib-5.0.5.tar.gz",
+    sha256: "e3b05c0a8ba1125d4dd560b36ab245314f9a89d5df67b72a856a8e4cc0f2b8f4" }
+]
+
+build_depends [ { package: "build-essential" } ]
+depends       [ { os: "linux" } ]
+provides      [ { package: "imagemagick" },
+                { package: "imagemagick-6" },
+                { package: "imagemagick-6.9" },
+                { package: "imagemagick-6.9.6" } ]
```
Comment (variadico) [inline packages/imagemagick-6.9.6-5.conf]: I noticed the package name is `imagemagick-6.9.6-5`. Is there also a `imagemagick-6.9.6-4`? I'm thinking we may want to add another provides field here:
```
{ package: "imagemagick-6.9.6-5" } ]
```

Comment (zquestz) [inline packages/imagemagick-6.9.6-5.conf]: +1

**Comment by variadico** [comment]
LGTM

File: packages/imagemagick-6.9.6-5.conf (hunk 1 above)
Comment (zquestz) [inline packages/imagemagick-6.9.6-5.conf]: PING. Please fix.

**Comment by zquestz** [comment]
LGTM once new package stanza is added.

File: packages/imagemagick-6.9.6-5.conf (hunk 1 above)
Comment (y0ssar1an) [inline packages/imagemagick-6.9.6-5.conf]: I just pushed the update. Thank you for the catch!

**Pull request merged**